ckanext/canada/schemas/presets.yaml

```diff
@@ -2802,11 +2835,45 @@ presets:
         en: Other
         fr: Autre
       value: other
+    required: true
+    form_include_blank_choice: true
     form_snippet: select.html
     display_snippet: select.html
-    # no 'scheming_required' in validators because this field may be omitted
-    # for non-related-item resources
-    validators: ignore_missing scheming_choices
+    validators: scheming_required scheming_choices
+    form_panel: relationships
+
+- preset_name: canada_resource_related_relationship
+  values:
+    field_name: related_relationship
+    label:
+      en: Relationship Type
+      fr: Type de relation
+    help_text:
+      en: The Related Record’s relationship with the resource
+      fr: Relation du dossier connexe avec le jeu de données
+    choices:
+    - label:
+        en: Defines
+        fr: FR Defines FR
+      value: defines
+    - label:
+        en: Defined by
+        fr: FR Defined par FR
+      value: defined_by
+    - label:
+        en: References
+        fr: FR References FR
+      value: references
+    - label:
+        en: Referenced by
+        fr: FR Referenced par FR
+      value: referenced_by
+    form_include_blank_choice: true
+    form_snippet: select.html
+    display_snippet: select.html
+    validators: scheming_required scheming_choices
+    required: true
+    form_panel: relationships
```

Comment:
is the form_panel stuff used anymore?

Reply:
It is in Resources template, but I think that the form_panel was mainly for the old related resources panel? So we can probably just not do panels? and then just call `super()` in our `resource_form.html`.

Reply:
hmmm the form_panels seem to be used to also hide and order a lot of the resource form fields. So just gunna leave it in for now